src/core/server/server.api.md

```diff
@@ -1907,9 +1965,33 @@ export interface SavedObjectsServiceSetup {
 export interface SavedObjectsServiceStart {
     createInternalRepository: (extraTypes?: string[]) => ISavedObjectsRepository;
     createScopedRepository: (req: KibanaRequest, extraTypes?: string[]) => ISavedObjectsRepository;
+    createSerializer: () => SavedObjectsSerializer;
     getScopedClient: (req: KibanaRequest, options?: SavedObjectsClientProviderOptions) => SavedObjectsClientContract;
 }
 
+// @public (undocumented)
+export interface SavedObjectsType {
+    convertToAliasScript?: string;
+    hidden: boolean;
+    indexPattern?: string;
+    mappings: SavedObjectsTypeMappingDefinition;
+    migrations?: SavedObjectMigrationMap;
+    name: string;
+    namespaceAgnostic: boolean;
+}
+
+// @public
+export interface SavedObjectsTypeMappingDefinition {
+    // (undocumented)
+    properties: SavedObjectsMappingProperties;
+}
+
+// @internal
+export interface SavedObjectsTypeMappingDefinitions {
```

Comment:
Is it necessary to expose this internal type? I coludn't see other types that reference it.

Reply:
No it's not. forgot to remove it when I cherrypicked from the `registerMappings` PR.